chunjun-assembly/pom.xml

```diff
@@ -33,15 +33,50 @@
 	<packaging>pom</packaging>
 	<name>ChunJun : Assembly</name>
 
-
 	<properties>
 		<skipTests>true</skipTests>
+		<!--version same as flink-1.12.7-bin-scala_2.12.tgz/lib/log4j-* -->
+		<log4j2.version>2.16.0</log4j2.version>
```

Comment:
version 2.17.0 may be better for it  fix CVE-2021-45105

Reply:
yes , 2.16.0 is the version flink 1.12.7 used , just same as it .

we(chunjun)  can update the version alone .